File: PERL/Cap05/c05ex08.pl
# c05ex08.pl

#!/usr/bin/perl

use strict;
use warnings;

my $RESP = "S";
do {
  printf "Entre um valor numerico: ";
  my $N = <STDIN>;
  chomp $N;

  my $R = $N * 3;

  print "Resultado = $R\n";
  print "\n";

  printf "Deseja continuar? (S/N): ";
  $RESP = <STDIN>;
  chomp $RESP;

  print "\n";  
} while (uc($RESP) eq "S");

print "\n";
print "Tecle <Enter> para encerrar... ";
<STDIN>;

File: RUBY/Cap05/c05ex08.rb
# c05ex08.rb

#!/usr/bin/env ruby

resp = "S"
begin
  print "Entre um valor numerico: "
  n = gets.chomp.to_i

  r = n * 3

  puts "Resultado = #{r}"
  puts

  print "Deseja continuar? (S/N): "
  resp = gets.chomp

  puts
end while (resp.upcase == "S")

puts
print "Tecle <Enter> para encerrar... "
gets
